packages/client/.eslintrc.cjs

```diff
@@ -6,8 +6,7 @@ module.exports = {
     project: 'packages/client/tsconfig.json'
   },
   rules: {
-    '@typescript-eslint/no-explicit-any': 'off',
-    '@typescript-eslint/ban-types': 'off',
     '@typescript-eslint/no-floating-promises': 'error',
+    "@typescript-eslint/strict-boolean-expressions": ["error", { allowNullableString: true, allowNullableObject: true }],
```

Comment:
Allow Nullable String is not safe for empty strings. Might revisit in the future.